packages/template-typescript-minimal/app/components/hello-javascript.jsx

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021, salesforce.com, inc.
+ * Copyright (c) 2023, salesforce.com, inc.
```

Comment:
```suggestion
 * Copyright (c) 2021, salesforce.com, inc.
```
Revert these changes in the unmodified files or use the correct entity in 2023:
```suggestion
 * Copyright (c) 2023, Salesforce, Inc.
```

Reply:
@wjhsf your proposal here violates linting rules as defined in `.eslintrc.js`

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/.eslintrc.js#L11